pom.xml

```diff
@@ -3294,6 +3294,12 @@
                 <param>
                   com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheckTest
                 </param>
+                <param>
+                  com.puppycrawl.tools.checkstyle.checks.AvoidEscapedUnicodeCharactersCheckTest
```

Comment:
Please add comment to make it clean why these tests are required. See comment for `AbstractJavadocCheckTest` for reference.

Reply:
Can you help me understand, what those numbers mean? Like removing `AbstractJavadocCheckTest` will result in coverage falling by those numbers in respective classes?

Reply:
yes. This information will be helpful for a possible refactoring of the CodePointUtil class. Someone may add a new method to CodePointUtil, then remove this test from the list and replace it with another test that will add full coverage of CodePointUtil.

Reply:
Since EmptyBlockCheck depends on utility functions defined in https://github.com/checkstyle/checkstyle/pull/11095/commits/59b2ff26fbc05e2c06640972969c003e098f3eae I think, there is no need to add `EmptyBlockCheckTest` in TargetTests, what do you think, (I tried running PItest removing EmptyBlockCheckTest from target tests and coverage and mutation was 100%, so should I remove EmptyBlockCheckTest and no comment needs to be added for AvoidEscaped... right?)

Reply:
Yes, please remove `EmptyBlockCheckTest`.

Reply:
done